src/main/scala/org/broadinstitute/dsde/workbench/sam/model/api/SamUserAllowances.scala

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.dsde.workbench.sam.model.api
+
+import spray.json.DefaultJsonProtocol._
+import spray.json.RootJsonFormat
+
+object SamUserAllowances {
+  implicit val SamUserAllowedResponseFormat: RootJsonFormat[SamUserAllowances] = jsonFormat2(SamUserAllowances.apply)
+
+  def apply(allowed: Boolean, enabled: Boolean, termsOfService: Boolean): SamUserAllowances =
+    SamUserAllowances(allowed, Map("enabled" -> enabled, "termsOfService" -> termsOfService))
+}
+final case class SamUserAllowances(
```

Comment:
or maybe `SamUserAllowance`?

Reply:
I like it being plural, since it details out all the reasons someone might not be allowed to use the system.